src/models/RevAiApiJob.ts

```diff
@@ -1,6 +1,10 @@
+import { JobStatus } from './async/JobStatus';
+import { JobType } from './JobType';
+
 export interface RevAiApiJob {
     id: string;
-    status: string;
+    status: JobStatus;
```

Comment:
is this backwards compatible? Like i.e if I compare the enum to a string, does it work? I would add a unit test to check for backwards compatibility

Reply:
Yes it is backwards compatible